### PR TITLE
Add RFMVasconcelos to OWNERS/approvers

### PR DESCRIPTION
**Pull request by thesuperzapper**
I would like to nominate @RFMVasconcelos as an approver for `kubeflow/website` in recognition of his significant contributions, and with him being the most active committer to the docs.

NOTE: the other diffs are removing some random spaces at the end of the lines

### CODE REVIEW

**Comment by thesuperzapper** [comment]
Can one of the existing approvers can OK this, (assuming your ok with it)?

@animeshsingh @Bobgy @joeliedtke @RFMVasconcelos

**Comment by 8bitmp3** [comment]
+1 @RFMVasconcelos has been excellent 👍

**Comment by RFMVasconcelos** [comment]
Thank you for the nomination @thesuperzapper! Very happy to keep adding value to the community :)

**Review by Jeffwan — APPROVED** [review]

**Comment by 8bitmp3** [comment]
Thanks @Jeffwan 

@RFMVasconcelos 😃  

image

**Comment by castrojo** [comment]
+1 So far Rui has been proposing and linking to all his proposals at the community meeting and has been working via issues to find consensus when needed, so there's been no surprises here. It would remove a bottleneck.

**Comment by Bobgy** [comment]
/lgtm
/approve

Thank you for the proposal! and I totally agree with adding @RFMVasconcelos as a root approver!

Rui raised the structure refactor proposal and drove substantial results. It's very challenging to reach consensus among the community with such a broad proposal.
What's more, Rui did most of the refactoring PRs by himself, helping the entire community to move forward to the agreed upon website structure.

As [discussed earlier](https://github.com/kubeflow/website/pull/1845#issuecomment-604127615) and expanding a little bit, I think a root OWNER should
* have already contributed to a broad and challenging effort benefitting the wider community
* be genuinely familiar with the repository and the setup
* be responsible for the entire website's quality
* know when to drive towards community consensus
* delegate permission of new folders / existing folders to the corresponding stakeholders

Therefore, I think granting root OWNER isn't only about the contributions, but also about meeting the other qualities that are required for the role.

Rui has clearly met these criteria, and I look forward to continuing collaborating with him in this new role.

**Comment by RFMVasconcelos** [comment]
Thank you for the nomination @thesuperzapper and for approving with kind words @Bobgy :) 

I very much wish to continue contributing to this project with all my ability! 🚀

**Pull request merged**
